src/main/java/dev/emi/trinkets/api/SlotReference.java

```diff
@@ -1,4 +1,7 @@
 package dev.emi.trinkets.api;
 
 public record SlotReference(TrinketInventory inventory, int index) {
+    public String getRefId() {
```

Comment:
Blank line before methods please, also I feel like this should probably also be `getId`, what do you think?

Reply:
getId is fine, just slightly confusing with a getId in a related, contained thing. But fine.

Reply:
Yeah I think `getId` is fine for now. I'll consider a further pass when I'm ready to merge that I can do on my own.